tools/fapi/tss2_nvread.c

```diff
@@ -93,13 +94,14 @@ int tss2_tool_onrun (FAPI_CONTEXT *fctx) {
             strlen(logData));
         if (r) {
             Fapi_Free (data);
+            Fapi_Free (logData);
             LOG_PERR ("open_write_and_close logData", r);
             return 1;
         }
     }
 
-    Fapi_Free (logData);
     Fapi_Free (data);
+    Fapi_Free (logData);
```

Comment:
why move this Fapi_Free? seems unneeded.

Reply:
Addressed in the latest commit.